### PR TITLE
Add topic for CSS Modules

### PR DESCRIPTION
**Pull request by david-fong**
### Thank you for contributing! Please confirm this pull request meets the following requirements:

- [x] I followed the contributing guidelines: https://github.com/github/explore/blob/master/CONTRIBUTING.md
- [x] I have no affiliation with the project I am suggesting (as a maintainer, creator, contractor, or employee).

### Which change are you proposing?

  - [ ] Suggesting edits to an existing topic or collection
  - [x] Curating a new topic or collection
  - [ ] Something that does not neatly fit into the binary options above

---------------------------------------------------------------------

### Curating a new topic or collection

- [x] I've formatted my changes as a new folder directory, named for the topic or collection as it appears in the URL on GitHub (e.g. `https://github.com/topics/[NAME]` or `https://github.com/collections/[NAME]`)
- [x] My folder contains a `*.png` image (if applicable) and `index.md`
- [x] All required fields in my `index.md` conform to the Style Guide and API docs: https://github.com/github/explore/tree/master/docs

> Modularity in programming languages can be used to improve the robustness of a system, and to prevent conflicts between dependencies. I believe that many developers would find CSS Modules useful- if only they knew that such a thing existed. I hope that creating this topic will help developers discover it.

### CODE REVIEW

**Review by iancanderson — CHANGES_REQUESTED** [review]
The logo looks a bit grainy. Could you find or generate a sharper looking image? Thanks!

File: topics/css-modules/index.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+---
+aliases: cssmodules
+# created_by: 
+display_name: CSS Modules
+github_url: https://github.com/css-modules/css-modules
+logo: css-modules.png
+related: css
+# released: May 25, 2015
```
[the remark this answers is not in the document]
Comment (david-fong) [reply topics/css-modules/index.md:7]: done!

File: topics/css-modules/index.md [hunk 2]
```diff
@@ -0,0 +1,13 @@
+---
+aliases: cssmodules
+display_name: CSS Modules
+github_url: https://github.com/css-modules/css-modules
+logo: css-modules.png
+related: css
+# released: May 25, 2015
```
Comment (iancanderson) [inline topics/css-modules/index.md:7]:
```suggestion
```

**Review by iancanderson — APPROVED** [review]
Thanks!

**Pull request merged**